packages/nextjs/src/index.server.ts

```diff
@@ -118,6 +118,11 @@ function addServerIntegrations(options: NextjsOptions): void {
   });
   integrations = addOrUpdateIntegration(defaultRewriteFramesIntegration, integrations);
 
+  const nativeBehaviourOnUncaughtException = new Integrations.OnUncaughtException();
+  integrations = addOrUpdateIntegration(nativeBehaviourOnUncaughtException, integrations, {
+    _options: { exitEvenIfOtherHandlersAreRegistered: false },
+  });
```

Comment:
I'm a little confused by this. The point of `addOrUpdateIntegration` is to say, "here's the integration to add if no instance is already there, but if an instance is there, update it with these options." This means that the integration to add should have the same options as the the ones which are being updated.

Reply:
Oh, I assumed the options didn't matter since the `UncaughtException` integration is part of the default integrations anyways but you're right - we should add the options in case the users override the integration.